src/core_plugins/metrics/public/components/aggs/derivative.js

```diff
@@ -84,7 +93,10 @@ export const DerivativeAgg = props => {
         <EuiFlexItem>
           <EuiFormRow
             id={htmlId('units')}
-            label="Units (1s, 1m, etc)"
+            label={(<FormattedMessage
+              id="tsvb.derivative.unitsLabel"
+              defaultMessage="Units (1s, 1m, etc)"
```

Comment:
Please add description `1s and 1m are required values and must not be translated.`